### PR TITLE
Simplify travis.yml file

### PR DESCRIPTION
**Pull request by tgalopin**
As discussed with @webmozart, the travis.yml file can be simplified.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,11 +21,7 @@ matrix:
     - php: nightly
   fast_finish: true
 
-install:
-  - composer self-update -q
-  - if [ -z "$deps" ]; then composer install -n; fi;
-  - if [ "$deps" = "low" ]; then composer update --prefer-lowest --prefer-stable --prefer-source -n; fi;
-  - if [ "$deps" = "high" ]; then composer update --prefer-source -n; fi;
+install: composer update $COMPOSER_FLAGS -n
```
Comment (webmozart) [inline .travis.yml]: This is missing the self-update, no?

Comment (GrahamCampbell) [inline .travis.yml]: no

Comment (GrahamCampbell) [inline .travis.yml]: You don't need to self-update?

Comment (webmozart) [inline .travis.yml]: Why was it in there before?

Comment (GrahamCampbell) [inline .travis.yml]: D'no. OCD?

Comment (tgalopin) [inline .travis.yml]: According to @stof and @GrahamCampbell, self-update is not required :) 

I think it's required only if we use features of lastest versions of Composer and TravisCI version is recent enough to support `--prefer-lowest` and `--prefer-stable`. It only adds useless build time.

Comment (webmozart) [inline .travis.yml]: :) ok

**Pull request merged**